config/default/manager_auth_proxy_patch.yaml

```diff
@@ -26,7 +26,8 @@ spec:
       - name: manager
         args:
         - "--health-probe-bind-address=:8081"
-        - "--metrics-bind-address=127.0.0.1:8080"
+        - "--diagnostics-address=127.0.0.1:8080"
+        - "--insecure-diagnostics"
```

Comment:
metrics-bind-address got deprecated in favour of diagnostics-address and insecure-diagnostics is needed to preserve the http metrics functionality from metrics-bind-address.